zebra-instructions.md

```diff
@@ -101,7 +101,7 @@ Install dependencies:
     sudo apt-get install redis
     redis-server
     ```
-2. Install and activate [`nodenv`](https://github.com/nodenv/nodenv#installation) or [`nvm`](https://github.com/nvm-sh/nvm#installing-and-updating)
+2. Install and activate a node version manager (e.g. [`nodenv`](https://github.com/nodenv/nodenv#installation) or [`nvm`](https://github.com/nvm-sh/nvm#installing-and-updating))
```

Comment:
Optional: Was there a specific reason you used nvm?

If nodenv didn't work for you, it would be good to document why.

Reply:
> Was there a specific reason you used nvm?

Purely popularity, I hadn't used nodenv before, but I tried it first and it worked well.